packages/project-integrity/src/fiori-project/index.ts

```diff
@@ -47,7 +68,8 @@ async function getFileList(projectRoot: string): Promise<string[]> {
  */
 async function getAdditionalStringContent(projectRoot: string): Promise<Content> {
     const capCustomPaths = await getCapCustomPaths(projectRoot);
-    return { capPaths: JSON.stringify(capCustomPaths) };
+    const cnsContent = await getCsnContent(projectRoot);
+    return { capPaths: JSON.stringify(capCustomPaths), csnPath: cnsContent };
```

Comment:
Why `csnPath`?
I'd name it just csn:

```suggestion
    const csn = await getCsnContent(projectRoot);
    return { capPaths: JSON.stringify(capCustomPaths), csn };
```

Reply:
covered in this patch https://github.com/SAP/open-ux-tools/pull/3096/commits/592c80dbcbcfdda7ece3814cb79723c3dbb65b8e